### DISCUSSION TITLE
Blog with posts in root directory

### DISCUSSION THREAD

**Discussion by davidmytton** [quarto-dev/quarto-cli#2180] [Q&A]
The standard blog behavior is to have [all posts in a subdirectory](https://quarto.org/docs/websites/website-blog.html#posts-directory) which the listing reads from to create the index of posts e.g. `example.com/posts/post-title`.

Is there a recommended method for having posts in the root directory e.g. `example.com/post-title/`? I understand I could just create new pages in the root, but then I believe that would lose the listing & RSS feed functionality? Or is the same limitation as discussed in https://github.com/quarto-dev/quarto-cli/discussions/1408 related to permalinks?

**Comment by dragonstyle** [comment]
Hi,

Listings are driven by their contents as a glob (<https://quarto.org/docs/reference/globs.html>) - you can point the listing at really any contents and it should successfully create a listing page and feed for that group of items. For example, to capture all `index.qmd` files that are in a subfolder one level below the current directory, you could do something like:

```
---
title: "Listing Example"
listing:
  contents:
    - "*/index.qmd"
---
```

I'm not sure about best practices for organization - but this should support just about any organization that you find productive.

HTH!
